coremltools/converters/mil/frontend/torch/load.py

```diff
@@ -41,6 +41,8 @@ def load(model_spec, debug=False, **kwargs):
         only.
     """
     torchscript = _torchscript_from_model(model_spec)
+    if torchscript.training:
+        _logging.warning("Model not in eval mode")
```

Comment:
Let's make this warning message actionable, i.e. let's tell them that they should probably call `.eval()` on their model before converting it.

I would change the error message to something like "Model is not in eval mode. Consider calling '.eval()' on your model prior to conversion".